engo_glfw.go

```diff
@@ -33,13 +33,43 @@ var (
 	canvasWidth, canvasHeight float32
 )
 
+// WindowResizeMessage is a message that's being dispatched whenever the game window is being resized by the gamer
+type WindowResizeMessage struct {
```

Comment:
Oh. Makes sense. I made a search and didn't found anything... I'll remove this part then!